packages/react-components/src/lib/cesium-map/map.tsx

```diff
@@ -117,6 +119,10 @@ export const CesiumMap: React.FC<CesiumMapProps> = (props) => {
     CesiumSceneModeEnum[] | undefined
   >();
   const [baseMaps, setBaseMaps] = useState<IBaseMaps | undefined>();
+  // eslint-disable-next-line
+  const [terrainProvider, setTerrainProvider] = useState<
```

Comment:
do you use **terrainProvider** somewhere?

Reply:
Da!